traces

```diff
@@ -37,6 +37,204 @@ const GITHUB_CONTRIBUTORS_POINT = 'https://api.github.com/repos/{repository}/con
+        'PrestaShop/ps_customtext',
+        'PrestaShop/ps_contactinfo',
+        'PrestaShop/ps_emailsubscription',
+        'PrestaShop/ps_banner',
+        'PrestaShop/ps_featuredproducts',
+        'PrestaShop/ps_imageslider',
+        'PrestaShop/ps_mainmenu',
+        'PrestaShop/ps_customersignin',
+        'PrestaShop/ps_categorytree',
+        'PrestaShop/ps_legalcompliance',
+        'PrestaShop/ps_wirepayment',
+        'PrestaShop/ps_shoppingcart',
+        'PrestaShop/ps_currencyselector',
+        'PrestaShop/ps_languageselector',
+        'PrestaShop/ps_facetedsearch',
+        'PrestaShop/ps_customeraccountlinks',
+        'PrestaShop/ps_searchbar',
+        'PrestaShop/ps_checkpayment',
+        'PrestaShop/ps_emailalerts',
+        'PrestaShop/ps_rssfeed',
+        'PrestaShop/ps_bestsellers',
+        'PrestaShop/ps_advertising',
+        'PrestaShop/ps_brandlist',
+        'PrestaShop/ps_categoryproducts',
+        'PrestaShop/ps_newproducts',
+        'PrestaShop/ps_specials',
+        'PrestaShop/ps_supplierlist',
+        'PrestaShop/ps_crossselling',
+        'PrestaShop/ps_productinfo',
+        'PrestaShop/ps_dataprivacy',
+        'PrestaShop/ps_cashondelivery',
+        'PrestaShop/ps_carriercomparison',
+        'PrestaShop/ps_viewedproduct',
+        'PrestaShop/ps_feeder',
+        'PrestaShop/ps_reminder',
+        'PrestaShop/ps_searchbarjqauto',
+        'PrestaShop/ps_buybuttonlite',
+        'PrestaShop/ps_faviconnotificationbo',
+        'PrestaShop/ps_healthcheck',
+        'PrestaShop/ps_qualityassurance',
+        'PrestaShop/ps_emailgenerator',
+        'PrestaShop/pstagger',
+        'PrestaShop/ps_emailsmanager',
+        'PrestaShop/ps_livetranslation',
+        'PrestaShop/ps_googleanalytics',
+        'PrestaShop/ps_themecusto',
+    ],
+    'tools' => [
+        'PrestaShop/PrestaShop-webservice-lib',
+        'PrestaShop/module-generator',
+        'PrestaShop/live-demo-devices',
+        'PrestaShop/docker',
+        'PrestaShop/docker-ci',
+        'PrestaShop/azure-template-performance',
+        'PrestaShop/azure-template-high-performance',
+        'PrestaShop/azure-template-basic',
+        'PrestaShop/docker-internal-images',
+        'PrestaShop/docker-templates',
+        'PrestaShop/TopContributors',
+        'PrestaShop/prestonbot',
+        'PrestaShop/issuebot',
+        'PrestaShop/traces',
+        'PrestaShop/travis-status-board',
+        'PrestaShop/prestashop-shop-creator',
+        'PrestaShop/mjml-theme-converter',
+        'PrestaShop/nightly-board',
+        'PrestaShop/github-webhook-parser',
+        'PrestaShop/presthubot-ui',
+        'PrestaShop/presthubot',
+        'PrestaShop/QANightlyResults',
+        'PrestaShop/core-weekly-generator',
+        'PrestaShop/email-templates-sdk',
+        'PrestaShop/fontmanager',
+        'PrestaShop/vagrant',
+    ],
+    'others' => [
+        'PrestaShop/prestashop.github.io',
+        'PrestaShop/solrsearch',
```

Comment:
is a module :thinking:

Reply:
is it something we should archive? does it work with 1.7? (and yes, I know this is not exactly the topic of this discussion, but knowing the answer would help to know what to do with it)

Reply:
Don't know, Don't know, should be seen with the @PrestaShop/product-team :)